Dockerfile

```diff
@@ -22,13 +22,9 @@ RUN tar cvf libs.tar --dereference --files-from=libs.txt
 # Minimal image
 #
 FROM martenseemann/quic-network-simulator-endpoint:latest
-# copy run scripts
+# copy run script
 COPY run_endpoint.sh .
 RUN chmod +x run_endpoint.sh
-COPY setup.sh .
```

Comment:
cool! yeah I guess we can also remove these from the repo. 
I think I added this early on..

Reply:
eh, you are indeed removing them